fix(ui): remove new badge from alert wizard

robinrendle opened the pull request:
Removing this new badge since this type of alert has been around for a while now.

![CleanShot 2022-03-16 at 09 53 45](https://user-images.githubusercontent.com/1900676/158644715-dad0478d-68d8-4d0c-b4c9-6f2a91da148b.png)

robinrendle committed:
fix(ui): remove new badge from alert wizard

robinrendle committed:
fix(ui): remove new badge from alert wizard

evanpurkhiser committed:
chore(deps): Remove unused csstype (#32678)

evanpurkhiser committed:
chore(deps): Bump mini-css-extract-plugin 2.0.0 -> 2.6.0 (#32683)

evanpurkhiser committed:
chore(deps): Bump react-grid-layout from 1.3.0 -> 1.3.4 (#32677)

scttcper commented:
need to remove the badge from the section definition as well https://github.com/getsentry/sentry/blob/master/static/app/views/alerts/wizard/options.tsx#L60

robinrendle committed:
removed featurebadge from types

robinrendle committed:
removed featurebadge from types

robinrendle committed:
removed featurebadge

robinrendle committed:
removed featurebadge

robinrendle committed:
removed component

robinrendle committed:
removed component

Commit:
ref(gsh): Change projects prop to memberProjects (#32658)

changed to memberProjects to be more descriptive of what the prop actually represents.

udameli committed:
feat(perf): Add base spans histogram endpoint (#32529)

* feat(perf): Add events spans histogram url

* feat(spans): Add spans histogram endpoint

* fix typos

* Add spans histogram query

* clean up the spans histogram query

* clean up test data

* make endpoint private

* add base tests

* udpate tests

* exclude spans histogram endpoint from the public list

* make endpoint private

* update codeowners

* use spans.exclusive_time in SnQL Function for spans histograms

* clear selected_columns

* document spans histogram query params

* handle quotes and backslashes in span ops

* rename args in span histogram snql function

* add proper tests

* add spans histogram function to the query builder

* more tests

robinrendle committed:
Merge branch 'rr/remove-alert-badge' of github.com:getsentry/sentry into rr/remove-alert-badge
